html/apps/forgotpassword/js/passwordrecovery.js

```diff
@@ -0,0 +1,32 @@
+$(document).ready(function()
+{
+    var emailerr = false, session = true;
+
+    $('#submit').click(function(){
+        checkEmpty();
+        if(emailerr){
+            alert("Please fill email");
+        } else{
+            var mail = encodeURIComponent($('#email').val());
+            alert("Email: " + mail + "\nSuccess, write post function");
```

Comment:
Please move to console log's and not alert's, too many alert's results in browsers complaining about suppressing the popups. We've spoken about this before